src/registration/index.ts

```diff
@@ -62,11 +72,11 @@ export default class RegistrationManager extends ExtrinsicBaseClass {
   /**
    * Registers a user with the given parameters.
    *
-   * @param program_pointer - Optional. Initial program associated with the user.
-   * @param keyVisibility - Key visibility level ('Public', 'Private'). Defaults to 'Public'.
-   * @param programDeployer - Account authorized to modify programs on behalf of the user.
+   * @param {RegistrationParams} params - The registration parameters.
+   * @param {ProgramInstance[]} params.programData - The initial program data associated with the user.
+   * @param {SS58Address} [params.programDeployer] - Optional. The account authorized to modify programs on behalf of the user.
```

Comment:
these should be removed / de-duplicated, because `RegistrationParams` interface/type should be source of truth?
```suggestion
   * @param {ProgramInstance[]} params.programData - The initial program data associated with the user.
   * @param {SS58Address} [params.programDeployer] - Optional. The account authorized to modify programs on behalf of the user.
```